db/migrate/20160708235959_add_web_hook_event.rb

```diff
@@ -0,0 +1,18 @@
+class AddWebHookEvent < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp' unless extension_enabled?('uuid-ossp')
+
+    create_table :web_hook_events, id: :uuid, default: 'uuid_generate_v1mc()' do |t|
```

Comment:
I expect several web hooks may work at the same time with a long period. `UUID` may be easier to distinguish with each others. It can also be generated in parallel